android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -18,6 +18,7 @@
 import android.telephony.TelephonyManager;
 import android.text.format.Formatter;
 import android.app.ActivityManager;
+import android.provider.Settings.Global;
```

Comment:
Wouldn't this import cause a problem for devices < Build.VERSION_CODES.JELLY_BEAN_MR1 ?